app/controllers/registration/RegistrationController.scala

```diff
@@ -133,6 +137,21 @@ class RegistrationController @Inject()(
       )
   }
 
+  // TODO check IV enabled flag here
```

Comment:
you haven't done your TODO

Reply:
yeah - was going to do as part of the uplift ticket

Reply:
i guess it doesn't really matter because the flag is checked here: `IdentityVerification.startIv()`

Reply:
It would be better placed in RegistrationController, but it would impact tests so leaving that until next time